manifests/init.pp

```diff
@@ -8,6 +8,7 @@
   String $service_name,
   Variant[String,Undef] $package_name,
   String $control_cmd,
+  String $control_setup_cmd,
```

Comment:
If an empty string is not valid (I guess it is not), better use:
```suggestion
  String[1] $control_setup_cmd,
```

We can also use the type form stdlib if the full path is required:

```suggestion
  Stdlib::Absolutepath $control_setup_cmd,
```